### ISSUE TITLE
How do you select the batches contained the sensitive data?

### ISSUE THREAD

**Issue by Brownchen** [lmgraves/AmnesiacML#1]
Thank you for publishing the code. It's a great attempt for machine unlearning. By the way, how do you select the sensitive data during the unlearning process and I can't find the corresponding code in  'Amnesiac ML Evaluation.ipynb'. I want to reproduce the model accuracy on the target and non-target data. Looking forward to your reply!

**Comment by lmgraves** [comment]
Hi! Thanks for reaching out. The sensitive data can be selected any way you
choose - for the CIFAR and MNIST domains I selected an entire class and
then had two test dataloaders, one including the sensitive class test data
and one including the rest of the test data. To test the membership
inference attack success I used a small random selection of training
instances as sensitive data.

The Amnesiac ML Evaluation.ipynb file is the testing that we did for figure
7 in the paper, but looking at it right now it's a big buggy. I'll fix it
later today and let you know when it's updated.

Thanks!

-Laura Graves

The github isn't the entire code base that we used, I hadn't included a
full testing and training cycle

**Comment by Brownchen** [comment]
Thank you for your reply! I do have some problems with the implementation
of trainloder and testloader.
 Could you please publish the entire code or just send it to my email <
brownchen7s@gmail.com> ?

Thanks a lot!

Laura Graves <notifications@github.com> 于2020年11月25日周三 下午10:01写道：

**Comment by lmgraves** [comment]
Hi, I've updated the evaluation notebook as well as added a new notebook
that tests the naive retraining, unlearning, and amnesiac unlearning
methods on resnet18 and CIFAR100. Please let me know if you are having
trouble testing it. Thanks!
-Laura Graves

**Comment by Brownchen** [comment]
Thank you so much!

**Comment by Chrisable** [comment]
What is the testtargetmodel in r, m = testtargetmodel()?
I can't find the corresponding definition.
Is this the complete code?
![image](https://user-images.githubusercontent.com/49330390/159665663-c1ab8368-7ec1-4378-9805-65890c02dea4.png)
